### PR TITLE
Checkpoint moving a lot of `graph` into `ingestor` who uses it the most.

### PR DESCRIPTION
**Pull request by bobmcwhirter**

### CODE REVIEW

**Comment by bobmcwhirter** [comment]
Yes, it's large and I apologize. And actually churned up more things I'd like to clean up, but outside of the scope of *this* particular PR.

File: modules/ingestor/src/graph/sbom/mod.rs [hunk 1]
```diff
@@ -44,7 +43,7 @@ pub struct SbomInformation {
 }
 
 impl From<()> for SbomInformation {
```
Comment (jcrossley3) [inline modules/ingestor/src/graph/sbom/mod.rs:45]: This seems weird. Where/why is this used?

File: modules/ingestor/src/graph/sbom/tests.rs [hunk 2]
```diff
@@ -45,7 +44,7 @@ async fn query_sboms() -> Result<(), anyhow::Error> {
         )
         .await?;
 
-    let other_sbom = system
+    let _other_sbom = system
```
Comment (jcrossley3) [inline modules/ingestor/src/graph/sbom/tests.rs:47]: Do we have to assign these? Seems like a smell.

File: modules/storage/Cargo.toml [hunk 3]
```diff
@@ -5,9 +5,9 @@ edition = "2021"
 publish = false
 
 [dependencies]
-trustify-common = { path = "../../common" }
-trustify-entity = { path = "../../entity" }
-trustify-module-graph = { path = "../graph" }
+trustify-common = { workspace = true }
+trustify-entity = { workspace = true }
+#trustify-module-fetch = { path = "../fetch" }
```
Comment (jcrossley3) [inline modules/storage/Cargo.toml:10]: Did you mean for this to remain?

**Review by jcrossley3 — APPROVED** [review]
Really too big to review, but I trust you.

File: modules/storage/Cargo.toml (hunk 3 above)
Comment (bobmcwhirter) [reply modules/storage/Cargo.toml:10]: Nope!

File: modules/ingestor/src/graph/sbom/tests.rs (hunk 2 above)
Comment (bobmcwhirter) [reply modules/ingestor/src/graph/sbom/tests.rs:47]: also, just carried over, except Clippy started to complain. I dunno why.  Took the easy path to quiet Clippy.

**Pull request merged**